src/Compilers/CSharp/Portable/CSharpExtensions.cs

```diff
@@ -1619,6 +1619,17 @@ public static Conversion ClassifyConversion(this SemanticModel? semanticModel, i
             var csmodel = semanticModel as CSharpSemanticModel;
             return csmodel?.GetDeclaredSymbol(node, cancellationToken);
         }
+
+        /// <summary>
+        /// Given a local function declaration syntax, get the corresponding symbol.
+        /// </summary>
+#pragma warning disable RS0026
```

Comment:
Why is this suppression needed here?

Reply:
I am not sure, since I don't see how this method is an overload. However I saw the same suppression here: https://github.com/dotnet/roslyn/blob/main/src/Compilers/CSharp/Portable/CSharpExtensions.cs#L1371